src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -933,6 +933,11 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		// Use for amend the label if inputData is not defined
 		if (resultSequence === undefined) {
 			let completionText = typeof completion.label === 'string' ? completion.label : completion.label.label;
+			
+			// Strip star prefix from IntelliCode starred suggestions when using label as completion text
+			if (completionText.startsWith('★ ')) {
```

Comment:
Not sure if this is the right way to handle this. Since Im getting:
`src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts(939,62): Unexpected unicode character: "★" (charCode: 9733). To suppress, use // allow-any-unicode-next-line`

Reply:
You can use the \u syntax to avoid that warning